blockdev/src/blockdev.rs

```diff
@@ -311,82 +491,42 @@ pub fn parse_size_mib(mut s: &str) -> Result<u64> {
-        }
+    fn test_loopback_cleanup_helper_spawn() {
+        // Test that we can spawn the cleanup helper process
+        // This test doesn't require root privileges and just verifies the spawn mechanism works
+        
+        // Create a temporary file to use as the "device"
+        let temp_file = NamedTempFile::new().unwrap();
+        let device_path = temp_file.path().to_string_lossy().to_string();
+        
+        // Try to spawn the cleanup helper
+        let result = LoopbackDevice::spawn_cleanup_helper(&device_path);
+        
+        // The spawn should succeed (though the helper will exit quickly since parent doesn't exist)
+        assert!(result.is_ok());
+        
+        // Clean up the temp file
+        drop(temp_file);
     }
 
     #[test]
     fn test_parse_lsblk() {
-        let fixture = include_str!("../tests/fixtures/lsblk.json");
-        let devs: DevicesOutput = serde_json::from_str(&fixture).unwrap();
-        let dev = devs.blockdevices.into_iter().next().unwrap();
-        let children = dev.children.as_deref().unwrap();
-        assert_eq!(children.len(), 3);
-        let first_child = &children[0];
-        assert_eq!(
-            first_child.parttype.as_deref().unwrap(),
-            "21686148-6449-6e6f-744e-656564454649"
-        );
-        assert_eq!(
-            first_child.partuuid.as_deref().unwrap(),
-            "3979e399-262f-4666-aabc-7ab5d3add2f0"
-        );
-    }
-
-    #[test]
-    fn test_parse_sfdisk() -> Result<()> {
-        let fixture = indoc::indoc! { r#"
-        {
-            "partitiontable": {
-               "label": "gpt",
-               "id": "A67AA901-2C72-4818-B098-7F1CAC127279",
-               "device": "/dev/loop0",
-               "unit": "sectors",
-               "firstlba": 34,
-               "lastlba": 20971486,
-               "sectorsize": 512,
-               "partitions": [
-                  {
-                     "node": "/dev/loop0p1",
-                     "start": 2048,
-                     "size": 8192,
-                     "type": "9E1A2D38-C612-4316-AA26-8B49521E5A8B",
-                     "uuid": "58A4C5F0-BD12-424C-B563-195AC65A25DD",
-                     "name": "PowerPC-PReP-boot"
-                  },{
-                     "node": "/dev/loop0p2",
-                     "start": 10240,
-                     "size": 20961247,
-                     "type": "0FC63DAF-8483-4772-8E79-3D69D8477DE4",
-                     "uuid": "F51ABB0D-DA16-4A21-83CB-37F4C805AAA0",
-                     "name": "root"
-                  }
-               ]
-            }
-         }
-        "# };
-        let table: SfDiskOutput = serde_json::from_str(&fixture).unwrap();
-        assert_eq!(
-            table.partitiontable.find("/dev/loop0p2").unwrap().size,
-            20961247
-        );
-        Ok(())
+        let data = fs::read_to_string("tests/fixtures/lsblk.json").unwrap();
```

Comment:
This change looks unrelated? I mean it's probably fine to do but let's break it into a separate commit

Reply:
This still needs fixing, can you move this section of the change to a different PR?